DESCRIPTION

```diff
@@ -1,14 +1,14 @@
 Package: drat
 Type: Package
 Title: 'Drat' R Archive Template
-Version: 0.1.6
-Date: 2020-05-29
+Version: 0.2.0
+Date: 2020-06-05
 Author: Dirk Eddelbuettel with contributions by Carl Boettiger, Neal Fultz,
  Sebastian Gibb, Colin Gillespie, Jan Górecki, Matt Jones, Thomas Leeper,
- Steven Pav, Jan Schulz, Christoph Stepper and Felix G.M. Ernst. 
+ Steven Pav, Jan Schulz, Christoph Stepper, Felix G.M. Ernst and Patrick Schratz. 
 Maintainer: Dirk Eddelbuettel <edd@debian.org>
 Depends: R (>= 3.2.0)
-Imports: utils
+Imports: utils, stringr
```

Comment:
We need clear motivation for expanding dependencies.  This does not qualify.

Reply:
`stringr` is a heavy dependency, such change would make `drat` unusable on many lightweight configurations.